Error in file
Exception type: NameError
Exception message: name 'a_test' is not defined
Stack trace:
File: Submissions/14027270.py
Line: 35
Func.Name: <module>
Message: y_predict = knc.predict(a_test)

